mirage/psf/psf_library.py

```diff
@@ -6,118 +6,112 @@
-            attribute of webbpsf.INSTR(). Spelling/capitalization must match what
-            webbpsf expects. See also special case for NIRCam. Default is "all"
+            the default "all" will run through all the detectors for that instrument.
+            Spelling/capitalization must match what WebbPSF expects. See also special
+            case for NIRCam. Default is "all".
 
-        num_psfs: int
+        num_psfs : int
             The total number of fiducial PSFs to be created and saved in the files. This
-            number must be a square number. Default is 16.
-            E.g. num_psfs = 16 will have the class create a 4x4 grid of fiducial PSFs.
+            must be a square number. Default is 16. E.g. num_psfs = 16 will create a 4x4
+            grid of fiducial PSFs.
 
-        psf_location: tuple
+        psf_location : tuple
             If num_psfs = 1, then this is used to set the (y,x) location of that PSF.
-            Default is (1024,1024).
+            Default is (1023,1023).
 
-        add_distortion: bool
+        add_distortion : bool
             If True, the PSF will have distortions applied: the geometric distortion from
             the detectors (using data from SIAF) and the rotation of the detectors with
             respect to the focal plane. Default is True.
 
-        fov_pixels: int
+        fov_pixels : int
             The field of view in undersampled detector pixels used by WebbPSF when
             creating the PSFs. Default is 101 pixels.
 
-        oversample: int
+        oversample : int
             The oversampling factor used by WebbPSF when creating the PSFs. Default is 5.
 
-        opd_type: str
+        opd_type : str
             The type of OPD map you would like to use to create the PSFs. Options are
             "predicted" or "requirements" where the predicted map is of the expected
             WFE and the requirements map is slightly more conservative (has slightly
-            larger WFE). Default is "requirements"
+            larger WFE). Default is "requirements".
 
-        opd_number: int
+        opd_number : int
             The realization of the OPD map pulled from the OPD file. Options are an
             integer from 0 to 9, one for each of the 10 Monte Carlo realizations of
             the telescope included in the OPD map file. Default is 0.
 
-        save: bool
+        save : bool
             True/False boolean if you want to save your file
 
-        fileloc: str
+        fileloc : str
             Where to save your file if "save" keyword is set to True. Default of None
             will save in the current directory
 
-        filename: str
+        filename : str
             The name to save your current file under if "save" keyword is set to True.
-            Default of None will save it in the form: "INSTR_FILT_fovp####_samp#_npsf##.fits"
+            Default of None will save it in the form: "{intr}_{filt}_fovp{}_samp{}_npsf{}.fits"
```

Comment:
I'm not sure if this was intentional or not, but the designation for "instrument" has changed from `INSTR` in the old version to `intr`.